src/opencost_parquet_exporter.py

```diff
@@ -69,89 +91,54 @@ def get_config(
     if s3_bucket is None:
         s3_bucket = os.environ.get('OPENCOST_PARQUET_S3_BUCKET', None)
     if file_key_prefix is None:
-        file_key_prefix = os.environ.get('OPENCOST_PARQUET_FILE_KEY_PREFIX', '/tmp/')
+        # TODO: Discuss: Format guideline?
+        file_key_prefix = os.environ.get(
+            'OPENCOST_PARQUET_FILE_KEY_PREFIX', '/tmp/')
     if aggregate_by is None:
-        aggregate_by = os.environ.get('OPENCOST_PARQUET_AGGREGATE', 'namespace,pod,container')
+        aggregate_by = os.environ.get(
+            'OPENCOST_PARQUET_AGGREGATE', 'namespace,pod,container')
     if step is None:
         step = os.environ.get('OPENCOST_PARQUET_STEP', '1h')
+    if storage_backend is None:
+        storage_backend = os.environ.get(
+            'OPENCOST_PARQUET_STORAGE_BACKEND', 'aws')  # For backward compatibility
 
     if s3_bucket is not None:
         config['s3_bucket'] = s3_bucket
+    config['storage_backend'] = storage_backend
     config['url'] = f"http://{hostname}:{port}/allocation/compute"
     config['file_key_prefix'] = file_key_prefix
+
+    # Azure-specific configuration
+    if config['storage_backend'] == 'azure':
+        config.update({
+            # pylint: disable=C0301
+            'azure_storage_account_name': os.environ.get('OPENCOST_PARQUET_AZURE_STORAGE_ACCOUNT_NAME'),
+            'azure_container_name': os.environ.get('OPENCOST_PARQUET_AZURE_CONTAINER_NAME'),
+            'azure_tenant': os.environ.get('OPENCOST_PARQUET_AZURE_TENANT'),
+            'azure_application_id': os.environ.get('OPENCOST_PARQUET_AZURE_APPLICATION_ID'),
+            'azure_application_secret': os.environ.get('OPENCOST_PARQUET_AZURE_APPLICATION_SECRET'),
+        })
+
     # If window is not specified assume we want yesterday data.
     if window_start is None or window_end is None:
         yesterday = datetime.strftime(
             datetime.now() - timedelta(1), '%Y-%m-%d')
         window_start = yesterday+'T00:00:00Z'
         window_end = yesterday+'T23:59:59Z'
     window = f"{window_start},{window_end}"
+    config['window_start'] = window_start
     config['params'] = (
         ("window", window),
         ("aggregate", aggregate_by),
         ("includeIdle", "false"),
         ("idleByNode", "false"),
         ("includeProportionalAssetResourceCosts", "false"),
         ("format", "json"),
-        ("step", step)
+        ("step", step),  # TODO: make this optional
+        ("accumulate", "true")  # TODO: Make this one a variable
```

Comment:
This was not set before, it is better if we keep the previous behaviour and parametrize this in case you need to set it to true to your usecase.